server/routes/api/auth.js

```diff
@@ -7,7 +7,7 @@ import UserController from '@controllers/users';
 const authRouter = express.Router();
 
 authRouter.post('/login', trim, UserController.login);
-authRouter.post('/activate_user', authorizeAccess, UserController.verifyAccount);
+authRouter.get('/activate_user', UserController.verifyAccount);
```

Comment:
why do you believe this should be a get request?

Reply:
A user is clicks on this link from their mail and it opens up directly in the browser. A post request might not open up this way.

Reply:
uhm, 
Firstly, that's not an actual reason

Also, when the user opens the link from the mail, he should connect to the frontend (not the backend). The frontend would then communicate with the backend.